app/models/request_type/validator.rb

```diff
@@ -31,6 +31,35 @@ def to_a
     delegate :to_sentence, to: :to_a
   end
 
+  class FlowcellTypeValidator # rubocop:todo Style/Documentation
+    attr_reader :request_type_id
+
+    def initialize(request_type_id)
+      @request_type_id = request_type_id
+    end
+
+    def request_type
+      RequestType.find(request_type_id)
+    end
+
+    def include?(option)
+      request_type
+        .flowcell_types
+        .where(requested_flowcell_type: option)
+        .pluck(:requested_flowcell_type)
+        .include?(option)
+    end
```

Comment:
I know this is following the pattern above, but it could probably be improved.

A drop in replacement would be:
```
    def include?(option)
      request_type
        .flowcell_types
        .exists?(requested_flowcell_type: option)
    end
```
https://api.rubyonrails.org/v6.0.4/classes/ActiveRecord/FinderMethods.html#method-i-exists-3F

Which avoids the need to instantiate an array, and simplifies the SQL generated.

It may be that we could get more improvements by caching the list of flowcell types, but this would likely require changes elsewhere in the code, so isn't worth worrying about at the moment.